internal/util/util_test.go

```diff
@@ -0,0 +1,75 @@
+package util
+
+import "testing"
+
+func TestGetStandardImageTag(t *testing.T) {
+
+	assertCorrectMessage := func(t testing.TB, got, want string) {
+		t.Helper()
+		if got != want {
+			t.Errorf("got %q want %q", got, want)
+		}
+	}
+
+	testArray := []string{
```

Comment:
This seems like a good place for a `[]struct` instead of an array of strings. You could have some thing like
```
struct case {
  desc string
  name string
  tag string
  expected string
}

testArray := []case {
  descr = "foo",
  name = "bar"
},{
  descr = "foo2",
  name = "bar2"
}
```
then instead of a for loop with using and index you can range over the `testArray` struct 
```
for _, case := range testArray {
 // checks using case.name or case.expected
}
```

excuse the syntax errors but this is a general idea

Reply:
That's a good idea, thanks @jmckulk . Updated, please take a look.